src/odemis/driver/autoscript_client.py

```diff
@@ -1519,7 +1534,7 @@ def _acquire(self) -> None:
                     # Retrieve the image (scans image, blocks until the image is received)
                     # TODO: use the metadata from the image acquisition _md once it's available
                     image, _md = self.parent.acquire_image(self._scanner.channel)
-
+                    image = ndimage.median_filter(image, 3)  # median filter to remove noise
```

Comment:
Why do you want to remove noise? I would expect the acquire method to just acquire and not do any image adjustments.